src/agent/solana.rs

```diff
@@ -86,8 +86,7 @@ pub mod network {
         logger: Logger,
     ) -> Result<Vec<JoinHandle<()>>> {
         // Publisher permissions updates between oracle and exporter
-        let (publisher_permissions_tx, publisher_permissions_rx) =
-            mpsc::channel(config.oracle.updates_channel_capacity);
```

Comment:
I believe this was the only use of the `updates_channel_capacity` config. Let's remove the config as well if that's the case.

Reply:
`updates_channel_capacity` is still used for the "updates" channel (`updates_tx`,`updates_rx`).